client/extensions/woocommerce/products/products-form/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import ProductsVariationTypesForm from '../products-variation-types-form';
+
+export default class ProductsForm extends Component {
+
+	static propTypes = {
+		product: PropTypes.shape( {
+			id: PropTypes.number.isRequired,
+			name: PropTypes.string.isRequired,
+			type: PropTypes.string.isRequired,
+		} )
+	};
+
+	render() {
+		return (
+			<Card>
+				<ProductsVariationTypesForm />
```

Comment:
Yeah that was why (We want the toggle / hint to always be displayed) -- but agreed it can be separate from the rest of the component.